assets/badge.js

```diff
@@ -75,8 +75,8 @@
 
       // update width
       var wp = 'slackin-width:' + id + ':'
-      if (wp === e.data.substr(0, wp.length)) {
-        var width = e.data.substr(wp.length)
+      if (wp === e.data.slice(0, wp.length)) {
```

Comment:
@emedvedev can you confirm the `slice` changes are OK? I'm going through the diff after 1.1.11

Reply:
Should be the same in this case, yes.

Reply:
I meant all of the `substr` to `slice` changes, just to be safe :)